packages/ketcher-core/package.json

```diff
@@ -48,6 +48,7 @@
     "d3": "^7.8.5",
     "lodash": "^4.17.21",
     "raphael": "^2.3.0",
+    "react-device-detect": "^2.2.2",
```

Comment:
I noticed that in your recent pull request, the dependency "react-device-detect" was added to the project. Could you please provide some context or reasons for including this dependency? I'd like to understand the purpose behind it.

Reply:
Sure, `shortcutsUtil.ts` new file from ketcher-core uses this library